File: cslavb/Csla/IDataPortalResult.vb
Public Interface IDataPortalResult

  ''' <summary>
  ''' Gets the resulting object.
  ''' </summary>
  ReadOnly Property [Object]() As Object

  ''' <summary>
  ''' Gets any resulting error information.
  ''' </summary>
  ReadOnly Property [Error]() As Exception

End Interface

File: cslacs/Csla/IDataPortalResult.cs
using System;

namespace Csla
{
  /// <summary>
  /// IDataPortalResult defines the results of DataPortal operation
  /// </summary>
  public interface IDataPortalResult
  {
    /// <summary>
    /// Gets the resulting object.
    /// </summary>
    object Object { get; }
    /// <summary>
    /// Gets any resulting error information.
    /// </summary>
    Exception Error { get; }
  }
}